hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/repository/PricesAndFeesRepository.java

```diff
@@ -0,0 +1,83 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+package com.hedera.mirror.web3.repository;
+
+import static com.hedera.mirror.web3.evm.config.EvmConfiguration.CACHE_MANAGER_1H;
+
+import com.hedera.mirror.common.domain.entity.Entity;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface PricesAndFeesRepository extends CrudRepository<Entity, Long> {
```

Comment:
There is no such PricesAndFees table
```suggestion
public interface FileDataRepository extends CrudRepository<FileData, Long> {
```